tests/src/Simple/Generics/Generics.cs

```diff
@@ -326,6 +328,53 @@ public static void Run()
         }
     }
 
+    class TestReflectionInvoke
+    {
+        struct Foo<T>
```

Comment:
Yup, and we have some of that covered in the Reflection test. I put this one in shared generics, because aside from testing the unbox stubs, this also tests the dictionary stuff.